cmd/_integration-tests/cli/cli_test.go

```diff
@@ -191,7 +179,7 @@ func TestAdditionalBindings(t *testing.T) {
 
 func testEndToEnd(defDir string, subcmd string, t *testing.T, trussOptions ...string) {
 	path := filepath.Join(basePath, defDir)
-	err := createTrussService(path)
+	err := createTrussService(path, trussOptions...)
```

Comment:
Wait... Was this never testing the `--pbout flag`? That is kinda bad... Oh well.

Reply:
It is kinda bad, though I did fix these tests on master to see if things where broken and they where not.